make each exhibit card have a single link to fix Accessibility complaint about missing ALT text on images

elrayle opened the pull request:
Fixes #2767

Moves the link outside the image and the title so there is only one link. This allows the role=presentation attribute on the image to allow for an empty alt attribute.

NOTE: First commit does not adjust indentations.  This makes it easier to see the code changes.  The second commit only modifies indentations.

cbeer reviewed (changes requested):
As I noted in https://github.com/projectblacklight/spotlight/issues/2767#issuecomment-866016690, I don't believe this is the expected behavior for decorative images.

elrayle commented:
@cbeer I saw your comment after I submitted the PR.  I am running this by our accessibility expert to see if he knows why Site Improve tagged this as an issue given your observations.

elrayle commented:
I moved the link outside the image and the title so there is only one link.  This allows the `role=presentation` attribute on the image to allow for an empty `alt` attribute.

elrayle commented:
@cbeer Can you take another look at this?  I believe that I addressed your comments and that this change is necessary to fix SiteImprove errors on the Home page.

![image](https://user-images.githubusercontent.com/6855473/127202303-5aece277-914d-4668-a2eb-28b1df0d8576.png)

elrayle commented:
The latest version of SiteImprove no longer sees this as an error.  For more information, see the [comment in Issue #2767](https://github.com/projectblacklight/spotlight/issues/2767#issuecomment-894280608). 
 
Closing.
